README.md

```diff
@@ -1,85 +1,81 @@
-DataStax Java Driver for Apache Cassandra
-=========================================
-
-.. image:: https://travis-ci.org/datastax/java-driver.svg?branch=2.0
-   :target: https://travis-ci.org/datastax/java-driver
-   :alt: Travis Build
-
+# Datastax Java Driver for Apache Cassandra
+
+*If you're reading this on GitHub, please note that this is the readme
```

Comment:
I wonder if this paragraph is still useful. Ultimately, documentation would only be generated, and thus available, for released versions, so there is no risk of seeing documentation for an unreleased version.